### PR TITLE
PR: Add missing `QtWebEngineWidgets.QWebEngineScript` import for PyQt5 and PySide2

### PR DESCRIPTION
**Pull request by EasyIsrael**

### CODE REVIEW

File: qtpy/QtWebEngineWidgets.py [hunk 1]
```diff
@@ -33,15 +34,20 @@
...
     from PyQt6.QtWebEngineCore import QWebEngineSettings
     from PyQt6.QtWebEngineCore import QWebEngineProfile
+    from PyQt6.QtWebEngineWidgets import QWebEngineScript
+        
 elif PYSIDE6:
     from PySide6.QtWebEngineWidgets import *
     from PySide6.QtWebEngineCore import QWebEnginePage
     from PySide6.QtWebEngineCore import QWebEngineSettings
     from PySide6.QtWebEngineCore import QWebEngineProfile
+    from PySide6.QtWebEngineWidgets import QWebEngineScript
```
Comment (CAM-Gerlach) [inline qtpy/QtWebEngineWidgets.py:44]: Same here...

**Comment by CAM-Gerlach** [comment]
FYI, I've solved the CI issues in #323 ; once that's merged, you can `git fetch upstream && git rebase upstream/master && git push -f` to get the fix here.

**Comment by EasyIsrael** [comment]
I have no pyside6 and no pyqt6 installed, so i just tested it on pyside2
and pyqt5.
Those two work now with my changes.
For the others i just added it the same way, because i thought it is
neccesary.



CAM Gerlach ***@***.***> schrieb am Sa., 26. Feb. 2022, 02:41:

File: qtpy/QtWebEngineWidgets.py (hunk 1 above)
Comment (CAM-Gerlach) [reply qtpy/QtWebEngineWidgets.py:44]:
```suggestion
```

**Review by dalthviz — COMMENTED** [review]
Thank you for your help with this @EasyIsrael ! Already merged the PR to fix our tests (https://github.com/spyder-ide/qtpy/pull/324) so you can go ahead and rebase to get the changes :+1: Also, besides @CAM-Gerlach comments regarding the removal of some lines added on PyQt6 and PySide6, it would be nice to add a new assertion when testing the module here: https://github.com/spyder-ide/qtpy/blob/1cd704f315ac44d91ad467b1757f42aa87d8a36e/qtpy/tests/test_qtwebenginewidgets.py#L4-L12

So adding the a line like:

```python
assert QtWebEngineWidgets.QWebEngineScript is not None
```

Let us know if you have any question or if you need help with the rebase or to do the changes

**Comment by CAM-Gerlach** [comment]
(As a reminder, the commands to rebase this:)

```bash
git fetch upstream
git rebase upstream/master
git push -f
```

**Comment by CAM-Gerlach** [comment]
Hey @EasyIsrael, thanks for bearing with us! It looks like things went wrong with Git here—seems like the original issue, which I should have noticed before, is that the PR was submitted from your `master` branch, rather than a feature branch with the changes. As a consequence things went haywire when you tried to pull in the CI fixes, with my changes are showing up on your PR, while the commit history shows multiple merges and the same commits multiple times, which not only is an issue here, but also will cause even more issues for you next time you try to submit a PR.

While you could `cherry-pick` the commits, since its only two lines in one file being changed, its probably easier to just copy them and redo them. So what I suggest to fix your repo is:

1. Copy (or just remember) the two lines you've changed.
2. Run the following to fix your `master` branch locally and remotely and set up a new feature branch:

   ```bash
   git fetch --all
   git reset --hard upstream/master
   git push -f
   git checkout -b add-qwebenginescript
   ```
3. Paste/type in the two changed lines
4. Run the following to commit and push (as normal):
   
   ```bash
   git commit -am "Add missing QWebEngineScript import for PyQt5 and PySide2"
   git push -u origin add-qwebenginescript
   ```
5. Create a new PR here off that branch and close this one

Thanks!

**Comment by EasyIsrael** [comment]
Thank you [CAM-Gerlach](https://github.com/CAM-Gerlach) and [dalthviz](https://github.com/dalthviz) for your help 
to push my changes to the qtpy repository.
It seems to be a bit more complicated. I'm not so familiar with git and the branch stuff. I will try to copy n paste my changes to the original master branch and make another pull request. I hope that works then.

**Comment by CAM-Gerlach** [comment]
Thanks, @EasyIsrael , looks like it did, so I'll close this one.

**Pull request closed without merging**